test/built-ins/Function/prototype/bind/get-fn-realm.js

```diff
@@ -7,17 +7,20 @@ description: >
     The realm of a bound function exotic object is the realm of its target
     function
 info: |
+    7.3.22 GetFunctionRealm ( obj )
+
     [...]
     2. If obj has a [[Realm]] internal slot, then
-       a, Return obj's [[Realm]] internal slot.
+       a. Return obj.[[Realm]].
     3. If obj is a Bound Function exotic object, then
-       a. Let target be obj's [[BoundTargetFunction]] internal slot.
+       a. Let target be obj.[[BoundTargetFunction]].
        b. Return ? GetFunctionRealm(target).
 features: [cross-realm]
 ---*/
 
 var other = $262.createRealm().global;
 var C = new other.Function();
-var B = Function.prototype.bind.call(C);
+C.prototype = null;
+var B = C.bind();
 
-assert.sameValue(Object.getPrototypeOf(new B()), C.prototype);
```

Comment:
Please note that this test does not invoke [`GetFunctionRealm`](https://tc39.github.io/ecma262/#sec-getfunctionrealm):

[`new`](https://tc39.github.io/ecma262/#sec-new-operator) calls bound function's [`[[Construct]]`](https://tc39.github.io/ecma262/#sec-bound-function-exotic-objects-construct-argumentslist-newtarget), which invokes regular [`[[Construct]]`](https://tc39.github.io/ecma262/#sec-ecmascript-function-objects-construct-argumentslist-newtarget) on target, which invokes [`GetPrototypeFromConstructor`](https://tc39.github.io/ecma262/#sec-getprototypefromconstructor) also on target.

The target, unlike the bound function, does have an object with non-enumerable `constructor` as its `prototype` property, so step 4 of [`GetPrototypeFromConstructor`](https://tc39.github.io/ecma262/#sec-getprototypefromconstructor)  won't be `true`.